lambdas/data-migration2/tests/test-executions.js

```diff
@@ -514,3 +515,38 @@ test.serial('migrateExecutions logs summary of migration for a specified logging
   await migrateExecutions(process.env, t.context.knex);
   t.true(logSpy.calledWith('Batch of 1 execution records processed, 1 total'));
 });
+
+test.serial('migrateExecutions calls storeError when system bucket is provided', async (t) => {
+  const key = `${process.env.stackName}/data-migration2-executions-errors.json`;
+
+  const parentExecution = fakeExecutionFactoryV2({
+    parentArn: undefined,
+    asyncOperationId: cryptoRandomString({ length: 10 }),
+  });
+  const childExecution = fakeExecutionFactoryV2({ parentArn: parentExecution.arn });
+
+  await Promise.all([
+    // Have to use Dynamo client directly because creating
+    // via model won't allow creation of an invalid record
+    dynamodbDocClient().put({
```

Comment:
I was curious and I took a quick look to find out if using `dynamodbDocClient().put` was really necessary. It is not. You can just use `executionsModel.create`. Furthermore, if the purpose of this test is just to ensure that an error is written for one failure, then we don't need the parent/child setup. Just creating an execution that tries to reference a non-existent async operation would be sufficient